README.md

```diff
@@ -35,14 +43,31 @@ fn main() {
 }
 ```
 
-
 ## Features
-Inspired by [Xilem](https://github.com/linebender/xilem), [Leptos](https://github.com/leptos-rs/leptos) and [rui](https://github.com/audulus/rui), Floem aims to be a high performance declarative UI library with minimal effort from the user. 
+
+Inspired by [Xilem](https://github.com/linebender/xilem), [Leptos](https://github.com/leptos-rs/leptos) and [rui](https://github.com/audulus/rui), Floem aims to be a high performance declarative UI library requiring minimal user effort.
+
 - **Cross-platform support**: Supports Windows, macOS and Linux with rendering using [Wgpu](https://github.com/gfx-rs/wgpu). In case a GPU is unavailable, a CPU renderer powered by [tiny-skia](https://github.com/RazrFalcon/tiny-skia) will be used.
-- **Fine-grained reactivity**: The entire library is built around reactive primitives inspired by [leptos_reactive](https://crates.io/crates/leptos_reactive). The reactive "signals" give the user a nice way to do reactive state management while maintaining very high performance.
-- **Performance**: The view tree is only run once, so the user can't accidentally put something expensive in the view generation function which slows down the whole application. The library also provides tools to help users write performant UI code. Check out the high performance [virtual list example](https://github.com/lapce/floem/tree/main/examples/virtual_list)
+- **Fine-grained reactivity**: The entire library is built around SolidJS-style reactive primitives inspired by [leptos_reactive](https://crates.io/crates/leptos_reactive). The reactive "signals" allow you to keep your UI up-to-date with minimal effort, all while maintaining very high performance.
```

Comment:
I might say fine grained reactive primitives instead of SolidJS-style.

Reply:
Replacing "SolidJS-style" with "fine-grained" would repeat a word that's already at the start of the line, so I suggest to remove "SolidJS-style". Would that work for you?

Reply:
Right I missed that. Have you personally compared the reactivity API to SolidJS? SolidJS-style seem to imply it's rather close, but I doubt it has leptos-like ownership scopes.

Reply:
I think mentioning solidjs makes sense because it’s the first one who does fine grained reactivity?

Reply:
Let's play it safe, I'll remove the mention of SolidJS.